s4e-backend/src/main/java/pl/cyfronet/s4e/data/repository/query/QueryTime.java

```diff
@@ -59,6 +65,23 @@ private void queryBySensingTime(Map<String, Object> params,
         }
     }
 
+    private void applyLicense(Map<String, Object> params,
+                              List<Object> parameters,
+                              StringBuilder resultQuery) {
+        if (params.containsKey(ACCESS_TYPE) && params.get(ACCESS_TYPE).equals(Product.AccessType.EUMETSAT)) {
+            LocalDateTime now = LocalDateTime.now();
```

Comment:
Don't access timestamp like this, it will make writing tests difficult, rather use Clock instance, as in EumetsatLicense among others